snapcraft/plugins/python2.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, name, options):
         super().__init__(name, options)
         self.stage_packages.extend([
             'python-dev',
+            'python-minimal',
```

Comment:
I'm afraid the fix for this one isn't so easy (fix doesn't work for me on xenial Classic Dimension). Perhaps the `env()` is used before `pluginhandler.pull()`?